decidim-conferences/spec/system/component_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Initiative Components" do
```

Comment:
Had to double check the filename 
```suggestion
describe "Conference Components" do
```